plutus-core/plutus-core/test/Spec.hs

```diff
@@ -198,6 +200,19 @@ testsGolden
     = testGroup "golden tests"
     . fmap (asGolden (format $ defPrettyConfigPlcClassic defPrettyConfigPlcOptions))
 
+testsDeBruijn :: [FilePath] -> TestTree
+testsDeBruijn
+    = testGroup "debruijn golden tests"
+    . fmap (\ fp -> goldenVsString (takeBaseName fp) (fp <.> "golden") $ deBruijnFP fp)
```

Comment:
you could use the `asGolden` helper, no?

Reply:
Also, these are tests going one way, we probably want to test the other way, which is where the issue was, right?